ai-assistant/src/helper/apihelper.tsx

```diff
@@ -267,6 +331,39 @@ export const handleActualApiRequest = async (
         });
       } catch (error) {
         console.log('Error in clusterRequest:', error);
+
+        // Check if this is a multi-container pod logs error
+        if (
+          isLogRequest(cleanedUrl) &&
+          error.message &&
+          error.message.includes('a container name must be specified')
+        ) {
+          // Extract pod name and available containers from error message
+          const podMatch = error.message.match(/for pod ([^,]+)/);
+          const containersMatch = error.message.match(/choose one of: \[([^\]]+)\]/);
+
+          if (podMatch && containersMatch) {
+            const podName = podMatch[1];
+            const containers = containersMatch[1].split(' ').map(c => c.trim());
+
+            const errorContent = `The pod "${podName}" has multiple containers. Please specify which container you want logs from:\n\nAvailable containers: ${containers.join(
+              ', '
+            )}\n\nTo get logs from a specific container, ask: "get logs from ${podName} container [container-name]"`;
+
+            aiManager.history.push({
+              error: false,
+              role: 'assistant',
+              content: errorContent,
+            });
+
+            return JSON.stringify({
+              error: false,
+              role: 'assistant',
+              content: errorContent,
+            });
```

Comment:
This is becoming a pattern. We need to see if we can optimize it.

Reply:
updated